src/renderers/native/ReactNativeEventEmitter.js

```diff
@@ -95,7 +95,7 @@ var ReactNativeEventEmitter = {
    *
    * @param {rootNodeID} rootNodeID React root node ID that event occurred on.
    * @param {TopLevelType} topLevelType Top level type of event.
-   * @param {object} nativeEventParam Object passed from native.
+   * @param {?object} nativeEventParam Object passed from native.
```

Comment:
Just wondering - how do we know that this third param is optional? Seems like it always gets passed in.

Reply:
Hi Flarnie, 

This one I could be wrong about it, I assumed it could be nullable due to the bellow line

```js
var nativeEvent = nativeEventParam || EMPTY_NATIVE_EVENT;
```

Reply:
That check does imply that - thinking we could update the Flow type here too in that case. Could be a follow-up PR.